src/Test/Tasty/MGolden.hs

```diff
@@ -51,8 +52,8 @@ instance IsTest Golden where
   run options golden _callback = runGolden golden options
   testOptions = pure . pure $ Option (Proxy :: Proxy Mode)
 
-goldenTest :: FilePath -> IO Text -> TestTree
-goldenTest expectedPath action = singleTest expectedPath Golden{..}
+goldenTest :: String -> FilePath -> IO Text -> TestTree
```

Comment:
I do not really like `tasty` is all `String` infected, also other parts of its API I do not agree with. I suspect at some point I'll have to make `mtasty` ;)